sections/01_Introduction.md

```diff
@@ -0,0 +1,7 @@
+# Introduction
+
+The Specification Paper is a community-driven initiative to provide a technical specification for Neo blockchain.
+This technical material is currently focused on NEO 3.0 documentation.
```

Comment:
We don't need to say in the technical specification what we are doing now. Simply write technical specifications.

Reply:
Agreed, Erik.